tfjs-layers/package.json

```diff
@@ -39,11 +39,13 @@
   "scripts": {
     "prep": "yarn install && yarn build",
     "build": "tsc",
+    "build-core": "cd ../tfjs-core && yarn && yarn build",
+    "build-core-ci": "cd ../tfjs-core && yarn && yarn build-ci",
     "build-npm": "./scripts/build-npm.sh",
```

Comment:
A suggestion: to defend against publishing layers with a linked core package, we should have `build-npm.sh` check that "@tensorflow/tfjs-core": points to a version and not "link:../tfjs-core" . Likewise for the other build-npm.sh scripts in the other packages.

Reply:
This is a reasonable suggestion for extra safety but in practice this won’t happen because of the release script :)